### ISSUE TITLE
How does the annealing work

### ISSUE THREAD

**Issue by pneumoman** [NVIDIA/flowtron#66]
In the paper it reads "We anneal the learning rate once the generalization error starts to plateau"  Is this done manually? If so can you quantify approximately what that that means?

**Comment by rafaelvalle** [comment]
We iteratively decrease the learning rate by half, observing the curve to decide when to decrease it.

**Comment by pneumoman** [comment]
Thank you!  Any ideas on the slope of loss at that point (I'm trying to automate it)  Unrelated would it be possible to add flow on the fly?

**Comment by rafaelvalle** [comment]
You can take a look at pytorch learning rate scheduler's to do it automatically (https://pytorch.org/docs/stable/optim.html)
It should be possible to a step of flow on the fly. You'd have to re-instantiate the Model and the Optimizer.

**Comment by pneumoman** [comment]
Thanks again!
